src/java.base/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -287,6 +291,16 @@ public final class SunEntries {
          * Set the digest provider based on whether native crypto is
          * enabled or not.
          */
+        /* Disabling OpenSSL usage in AIX due to perfomance regression observed */
```

Comment:
This sentence should end with a period; suggest "on AIX" instead of "in AIX"
```java
        /* Disable use of OpenSSL on AIX due to an observed perfomance regression. */
```

Reply:
Fixed.